<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1443</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_RampEditor">
       <attribute name="title">
        <string>Ramp Editor</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="RampEditor_mpl" native="true">
          <layout class="QVBoxLayout" name="RampEditor_mpl_vl"/>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="Qlist_PlotSelectorRamp">
             <item>
              <property name="text">
               <string>Momentum</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tune</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Chromaticity</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="StoneButtons" stretch="0,0,0">
             <item>
              <widget class="QPushButton" name="Up">
               <property name="text">
                <string>Up</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Down">
               <property name="text">
                <string>Down</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Copy">
               <property name="text">
                <string>Copy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="Qlist_StoneManipulator"/>
           </item>
           <item>
            <widget class="QListWidget" name="Qlist_GeneralRampParm"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_StoneEditor">
       <attribute name="title">
        <string>Stone Editor</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="StoneEditor_mpl" native="true">
          <layout class="QVBoxLayout" name="StoneEditor_mpl_vl"/>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="Qlist_PlotSelectorStone">
             <item>
              <property name="text">
               <string>Beta</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dispersion</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Phi</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Alpha</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="Qlist_QuadManipulator"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1443</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
